@kiva/kv-components/utils/loanUtils.js

```diff
@@ -76,6 +77,14 @@ export function getLendCtaSelectedOption(
 			// Top up campaign defaults to $5
 			return Number(unreservedAmount > 5 ? 5 : unreservedAmount).toFixed();
 		}
+
+		if (fiveDollarsSelected) {
```

Comment:
This isn't needed and will be covered by the following return statement.

Reply:
That's true, thanks!